include/Network/HTTP/BadGatewayException.php

```diff
@@ -9,6 +9,7 @@
 /**
  * @brief Bad Gateway (502) http exception
  */
-class BadGatewayException extends HTTPException {
+class BadGatewayException extends HTTPException 
+{
```

Comment:
I find it really ugly, since it totally breaks the convention with all the other control structure things.